app/views/layouts/_navbar.html.erb

```diff
@@ -9,14 +9,14 @@
 
   <div class="collapse navbar-collapse" id="myNavbar">
     <ul class="nav navbar-nav one-line">
-      <li> <%= link_to 'Home', home_path, class: 'navbar-brand' %> </li>
+      <li> <%= link_to 'Litter sn<span style="font-style:italic">App</span>'.html_safe, home_path, id: 'navbar-brand' %> </li>
```

Comment:
We'll skip this for now, but it's better to add a class and then add styles to that class in your css files.